Source/Flow/Public/Nodes/Route/FlowNode_NamedRerouteDeclaration.h

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Nodes/FlowNode.h"
+
+#include "FlowNode_NamedRerouteDeclaration.generated.h"
+
+/**
+ * 
+ */
+UCLASS(NotBlueprintable)
+class FLOW_API UFlowNode_NamedRerouteDeclaration : public UFlowNode
+{
+	GENERATED_UCLASS_BODY()
+	friend class UFlowNode_NamedRerouteUsage;
+	
+protected:
+	
+	virtual void ExecuteInput(const FName& PinName) override;
+	//~ Begin UObject Interface
+#if WITH_EDITOR
+	virtual void PostInitProperties() override;
+	virtual void PostLoad() override;
+	virtual void PostDuplicate(bool bDuplicateForPIE) override;
+	virtual bool GetDynamicTitleColor(FLinearColor& OutColor) const override;
+	virtual void PostEditChangeProperty(FPropertyChangedEvent& PropertyChangedEvent) override;
+#endif
+	virtual FText GetNodeTitle() const override;
+
+public:
+	UPROPERTY(EditAnywhere, Category = "Named Reroute")
+	FName NodeTitle;
```

Comment:
I would also consider wrapping properties NodeTitle, CustomNodeColor with #if WITH_EDITORONLY_DATA, because they are used (and should be used) in editor code